test/org/daisy/dotify/formatter/impl/writer/PEFMediaWriterTest.java

```diff
@@ -66,4 +71,43 @@ public void write(int b) throws IOException {
             + "</pef>";
         assertEquals(exp, w.toString().replaceAll("[\\r\\n]+", ""));
     }
+
+    @Test
+    public void testExternalReference() throws PagedMediaWriterException {
+        PEFMediaWriter p = new PEFMediaWriter(new Properties());
+
+        Map<QName, String> ref = new HashMap<>();
+        ref.put(new QName(null, "id", "example"), "TestValue");
```

Comment:
Note that this is an attribute without namespace, which isn't allowed in a PEF. So the PEF writer should probably ignore these (or raise an error).